dv/uvm/yaml/rtl_simulation.yaml

```diff
@@ -88,3 +88,22 @@
       -covbaserun test
     wave_opts: >
       -input <cwd>/ius.tcl
+
+- tool: riviera
+  env_var: ALDEC_PATH
+  compile:
+    cmd:
+      - "vlib <out>/work"
+      - "vlog -work <out>/work
+        +incdir+<ALDEC_PATH>/vlib/uvm-1.2/src
+        +incdir+${PRJ_DIR}/ibex/shared/rtl
+        -l uvm_1_2
+        -err VCP2694 W1
```

Comment:
Can you add a comment why this flag is needed/what it does?

Reply:
i've added the requested note. Maybe you can consider to fix the sources so this argument will not be neded